documentation/source/swagger/parts/schemas/loans_apis/LoansFinancingsContract.yaml

```
@@ -113,9 +113,11 @@ properties:
     type: string
     format: date
     maxLength: 10
-    pattern: ^(\d{4})-(1[0-2]|0?[1-9])-(3[01]|[12][0-9]|0?[1-9])$
+    pattern: ^(\d{4})-(1[0-2]|0?[1-9])-(3[01]|[12][0-9]|0?[1-9])$|^NA$
     example: "2018-02-15"
-    description: Data de vencimento primeira parcela do principal
+    description: |
+      Data de vencimento primeira parcela do principal
+      Caso a instituição não possua a informação para compartilhamento, informar NA
```

```suggestion
     [Restrição] Caso a instituição não possua a informação para compartilhamento, informar NA
```
@andretrindade Se eu não estiver errado, o script que gera os dicionários precisa de [Restrição] pra identificar a restrição no campo e colocar no dicionário